components/sendgrid/actions/get-a-block/get-a-block.mjs

```diff
@@ -0,0 +1,33 @@
+import validate from "validate.js";
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "sendgrid-get-a-block",
+  name: "Get a Block",
+  description: "Gets a specific block. [See the docs here](https://docs.sendgrid.com/api-reference/blocks-api/retrieve-a-specific-block)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    email: {
```

Comment:
I notice some repeatable uses of this variable through the actions. Maybe we can add it into `app`?